src/citadel/__snapshots__/citadel.controller.spec.ts.snap

```diff
@@ -2,13 +2,13 @@
 
 exports[`CitadelController GET /citadel/v1/rewards should fail without token addr param 1`] = `
 Object {
-  "message": "Error: Token should be specified",
+  "message": "Error: processing response error (body=\\"{\\\\\\"jsonrpc\\\\\\":\\\\\\"2.0\\\\\\",\\\\\\"error\\\\\\":{\\\\\\"code\\\\\\":-32600,\\\\\\"message\\\\\\":\\\\\\"Invalid Request. Requested data is older than 128 blocks.\\\\\\"},\\\\\\"id\\\\\\":51}\\", error={\\"code\\":-32600}, requestBody=\\"{\\\\\\"method\\\\\\":\\\\\\"eth_getLogs\\\\\\",\\\\\\"params\\\\\\":[{\\\\\\"fromBlock\\\\\\":\\\\\\"0xdfb49f\\\\\\",\\\\\\"toBlock\\\\\\":\\\\\\"0xdfe837\\\\\\",\\\\\\"address\\\\\\":\\\\\\"0xb1c38253ad6ab3e2a2d53a094692fcf1321b12d4\\\\\\",\\\\\\"topics\\\\\\":[\\\\\\"0xa3e98ad001ab3597c5c4a0afe6de30207ef7d124a67337acefc3d5d5d905242f\\\\\\"]}],\\\\\\"id\\\\\\":51,\\\\\\"jsonrpc\\\\\\":\\\\\\"2.0\\\\\\"}\\", requestMethod=\\"POST\\", url=\\"https://cloudflare-eth.com\\", code=SERVER_ERROR, version=web/5.6.0)",
```

Comment:
yeah so this was removed in the sdk and now :( so will need to be handled in @NamelsKing pr probably?